doc/user-guide/user-guide.rst

```diff
@@ -111,21 +111,22 @@ To deal with this process in a more comprehensive way, this repository uses a di
     .. attention::
 
       Make sure you select the correct cycle branch when opening a PR.
-      By default GitHub will assign the ``master`` branch as a target to all PRs.
+      By default GitHub will assign the ``main`` branch as a target to all PRs.
       You must manually select the cycle branch in this occasion.
 
-  - If new revisions to the current cycle (say cycle 20) are needed to support the production environment, those changes must be done on ticket branches and merged back to ``master``, as usual.
+  - If new revisions to the current cycle (say cycle 20) are needed to support the production environment, ticket branches out of ``main`` should be created and PRs must be opened and merged back to ``main``, as usual.
 
-  - If new revisions to the new cycle are needed, before the new cycle is officially released, ticket branches out of the new cycle branch should be created (e.g. ``cycle/0021``) and PRs must be opened and merged **back to the cycle branch**.
+  - If new revisions to the new cycle are needed **before the new cycle is officially released**, ticket branches out of the new cycle branch should be created (e.g. ``cycle/0021``) and PRs must be opened and merged **back to the cycle branch**.
```

Comment:
I think we need to add something saying that the revision should not be increased while the new revision is not officially released.

Reply:
Good point. I added another bullet explaining that.